code/modules/admin/fun_verbs.dm

```diff
@@ -572,13 +572,15 @@
 			var/heavy_impact_range = input("Heavy impact range (in tiles):", "Drop Bomb") as null|num
 			var/light_impact_range = input("Light impact range (in tiles):", "Drop Bomb") as null|num
 			var/flash_range = input("Flash range (in tiles):", "Drop Bomb") as null|num
-			if(isnull(devastation_range) || isnull(heavy_impact_range) || isnull(light_impact_range) || isnull(flash_range))
+			var/flame_range = input("Flame range (in tiles):", "Drop Bomb") as null|num
+			if(isnull(devastation_range) || isnull(heavy_impact_range) || isnull(light_impact_range) || isnull(flash_range) || isnull(flame_range))
 				return
 			devastation_range = CLAMP(devastation_range, -1, 10000)
 			heavy_impact_range = CLAMP(heavy_impact_range, -1, 10000)
 			light_impact_range = CLAMP(light_impact_range, -1, 10000)
 			flash_range = CLAMP(flash_range, -1, 10000)
-			explosion(usr.loc, devastation_range, heavy_impact_range, light_impact_range, flash_range)
+			flame_range = CLAMP(flame_range, -1, 10000)
```

Comment:
The maps are 255x255 tiles. This is a range value. Shouldn't these values be clamped at 255? Alternatively at `max(world.maxy, world.maxy)`